actors/runtime/src/main/java/cloud/orbit/actors/Stage.java

```diff
@@ -1255,11 +1255,8 @@ public boolean canActivateActor(final String interfaceName)
     {
         if (getState() != NodeCapabilities.NodeState.RUNNING)
         {
-            // todo, improve this
-            if (hosting.getServerNodes().size() > 1)
```

Comment:
tbh it sounds like this is in the wrong spot. If you decide to restart your nodes one by one you don't want this method returning true if it is in the middle of shutting down.